pages/Documents/GettingStarted/customizing-the-conversational-cloud.md

```diff
@@ -4,6 +4,8 @@ sitesection: Documents
 categoryname: "Getting Started"
 permalink: customizing-the-conversational-cloud.html
 indicator: both
+date: 2019-01-30 23:22:00 +0000
```

Comment:
(Let me know when ready to review! Quick questions:)

1. Can we rename these fields, or are we tied to “date” and “last_modified”?
2. Does this also work without a time, i.e. just using YYYY-MM-DD? (Preferred)

Reply:
hey its ready we can change those fields. let me know what are the fields and I'll change.

it is actually doesn't work in a different structure but I will test maybe there is a solution but as far as I tested it didn't work with only a date.